app/services/account_reset/delete_account.rb

```diff
@@ -66,13 +66,20 @@ def notify_user_via_email_of_deletion
     end
     # rubocop:enable IdentityIdp/MailLaterLinter
 
+    def profile_components
+      return nil if !user.identity_verified?
+      ProofingComponent.create_or_find_by(user: user)
```

Comment:
We don't want to create proofing components.

We should be able to pull them off the active profile.

```suggestion
      user.active_profile.proofing_components if user.active_profile?
```